analytics/analytics/core/models/facts.py

```diff
@@ -0,0 +1,166 @@
+from django.db import models
+
+from analytics.core.models.dimensions import (
+    DateDimension,
+    JobDataDimension,
+    NodeDimension,
+    PackageDimension,
+    PackageSpecDimension,
+    RunnerDimension,
+    TimeDimension,
+    TimerDataDimension,
+    TimerPhaseDimension,
+)
+
+
+class JobFact(models.Model):
+    # Foreign Keys
+    start_date = models.ForeignKey(DateDimension, on_delete=models.PROTECT)
+    start_time = models.ForeignKey(TimeDimension, on_delete=models.PROTECT)
+
+    node = models.ForeignKey(NodeDimension, on_delete=models.PROTECT)
+    runner = models.ForeignKey(RunnerDimension, on_delete=models.PROTECT)
+    package = models.ForeignKey(PackageDimension, on_delete=models.PROTECT)
+    spec = models.ForeignKey(PackageSpecDimension, on_delete=models.PROTECT)
+    job = models.ForeignKey(JobDataDimension, on_delete=models.PROTECT)
+
+    # ############
+    # Numeric Data
+    # ############
+
+    duration = models.DurationField()
+    duration_seconds = models.FloatField(
+        help_text="The duration of this job represented as seconds"
+    )
+
+    # Pod and node info (null if not run in cluster)
+    pod_node_occupancy = models.FloatField(null=True, default=None)
+    pod_cpu_usage_seconds = models.FloatField(null=True, default=None)
+    pod_max_mem = models.PositiveBigIntegerField(null=True, default=None)
+    pod_avg_mem = models.PositiveBigIntegerField(null=True, default=None)
+    node_price_per_second = models.DecimalField(
+        max_digits=9,
+        decimal_places=8,
+        help_text="The price per second (in USD) of the spot instance this job ran on, at the time of running.",
```

Comment:
:+1: 

FWIW, I think [Field.db_comment](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.Field.db_comment) will add it into postgres and *might* translate to a metabase description field somehow (https://github.com/metabase/metabase/pull/6974).

Reply:
I can confirm this is the case, thanks!